templates/webapi/main/group_builds_functionality_view.html.ep

```diff
@@ -0,0 +1,61 @@
+% for my $child (@{$children}) {
+  % my $hasJobs = 0;
```

Comment:
Please call this `$has_jobs` as it is a Perl variable.

Reply:
wip

Reply:
Fixed